content/en/user-guide/integrations/crossplane/index.md

```diff
@@ -0,0 +1,226 @@
+
+## Tutorial: Creating AWS resources in LocalStack via Crossplane
+
+In the following, we provide a step-by-step guide for installing Crossplane in a local test environment, and creating AWS resources (S3 bucket, SQS queue) in LocalStack via Crossplane.
+
+### Step 1: Installing Crossplane in local Kubernetes
+
+As a prerequisite, we need the following:
+* LocalStack running in local Docker
+* A local Kubernetes cluster:
+  * We can use the [embedded Kubernetes cluster](https://docs.docker.com/desktop/kubernetes) that ships with modern versions of Docker Desktop (can be easily enabled in the Docker settings)
+  * Alternatively, you can [create a local EKS cluster](https://docs.localstack.cloud/user-guide/aws/elastic-kubernetes-service/#create-an-embedded-kubernetes-cluster) in LocalStack directly, which will spin up a light-weight embedded `k3d` Kubernetes cluster in your Docker environment
+* The [`helm`](https://helm.sh) and [`kubectl`](https://kubernetes.io/docs/tasks/tools/#kubectl) command-line clients installed
+
+Once your `kubectl` is configured to point to the local Kubernetes cluster, we first install Crossplane via `helm`:
+{{<command>}}
+$ helm repo add crossplane-stable https://charts.crossplane.io/stable
+$ helm repo update
+$ helm install crossplane crossplane-stable/crossplane --namespace crossplane-system --create-namespace
+{{</command>}}
+
+The installation may take a few minutes. In parallel, we can install the `crossplane` command-line extensions for `kubectl`:
+{{<command>}}
+$ curl -sL https://raw.githubusercontent.com/crossplane/crossplane/master/install.sh | bash
+...
+$ sudo mv kubectl-crossplane /usr/local/bin
+{{</command>}}
+
+To confirm that the installation was successful, we can run these `kubectl` commands, which should yield output similar to the following:
+{{<command>}}
+$ kubectl crossplane --version
+v1.13.2
+
+$ kubectl get crds | grep crossplane
+compositions.apiextensions.crossplane.io                     2023-09-03T11:30:36Z
+configurations.pkg.crossplane.io                             2023-09-03T11:30:36Z
+...
+{{</command>}}
+
+### Step 2: Installing the Crossplane AWS Provider
+
+Once the basic Crossplane installation is running properly, we can proceed with installing the AWS provider.
+Newer versions of Crossplane promote the use of [provider families](https://docs.upbound.io/providers/provider-families), which are collections of providers for different groups of resources.
+For example, there is a separate provider for each individual AWS service (like S3, SQS, Lambda, etc), and in addition provider family provides shared resources for common configuration of all services (e.g., credentials, etc).
+
+In the following, we first install the AWS provider for S3.
+Please note that you can copy/paste the entire multi-line command below into your terminal:
```

Comment:
```suggestion
Note that you can copy/paste the entire multi-line command below into your terminal:
```

We should try to avoid words like "please" in technical docs.